server/src/test/java/org/opensearch/cluster/routing/remote/RemoteRoutingTableServiceTests.java

```diff
@@ -347,136 +352,13 @@ public void testGetIndicesRoutingMapDiffIndexDeleted() {
         assertEquals(indexName, diff.getDeletes().get(0));
     }
 
-    public void testGetIndexRoutingAsyncAction() throws IOException {
```

Comment:
Why are these tests removed?

Reply:
Tests are renamed into the appropriate function name.